src/Profile/EditProfile.js

```diff
@@ -273,13 +279,25 @@ class EditProfile extends React.Component {
               <br></br>
               <br></br>
 
-              <p>Profile Picture</p>
+              <p>Profile Picture</p><br></br>
+              {this.state.profilePicture 
+                && <img src={this.state.profilePicture}></img>
+              }
             </div>
+            <p >Update image:</p>
+                <input 
+                  type="file" 
+                  id="img" 
+                  name="img" 
+                  accept="image/*">
+                </input>
+                <p >Update image:</p>
             <div class="updateProfileButton">
               <input type="submit" value="Update" />
               {this.state.updatedMessage}
             </div>
           </form>
+          <button onClick = {this.removePicture}>Remove picture</button>
```

Comment:
Are you removing image from database if user choose to remove picture?

Reply:
yes it removes the picture when it is submit because the value within the state will be null and that will be sent to the backend.